### ISSUE TITLE
QuickReports Property for Uncommitted state

### ISSUE THREAD

**Issue by ahavas** [IBM/paxapi#21]
My team would like some type of visual indication that they edited a QR without committing changes.

Is there some type PAXAPI QR property or CustomProperty that I can programmatically access that indicates a Quick Report has uncommitted values, i.e. is in an uncommitted state?

**Comment by tedphillips** [comment]
i don't believe this is exposed, but it is something we can consider. thanks for the feedback, will bring it up in our internal discussions.
